src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -82,12 +77,7 @@ void fillInnerParts() {
         eateryListPanel = new EateryListPanel(logic.getFilteredEateryList());
         eateryListPanelPlaceholder.getChildren().add(eateryListPanel.getRoot());
 
-        ReadOnlyFeedList feedList = logic.getFeedList();
-        ObservableList<FeedPost> feedPostList = FXCollections.observableArrayList();
-        for (Feed feed : feedList.getFeedList()) {
-            feedPostList.addAll(feed.fetchPosts());
-        }
-        feedPostListPanel = new FeedPostListPanel(feedPostList);
+        feedPostListPanel = new FeedPostListPanel(logic.getFeedList());
```

Comment:
sick implementation to shift the collection of feeds into the FeedPostListPanel to speed up the start up of MainWindow